### DISCUSSION TITLE
Commitment to keeping Flatparse alive?

### DISCUSSION THREAD

**Discussion by liamzee** [AndrasKovacs/flatparse#46] [General]
Just curious, I'm working on a project that MIGHT have some of its code adapted for Cabal.

The problem with that is, if it gets adapted for Cabal, there'll be some maintenance questions as to whether flatparse will remain supported, as one of the libraries I'm using already had its maintainer admit he was unable to keep his library well-supported.

**Comment by AndrasKovacs** [comment]
Personally, I'm committed to maintenance.

**Comment by liamzee** [comment]
Thanks! I don't seem to have gotten opposition on using flatparse to parse Cabal files, so I am now the bottleneck. :)

**Comment by raehik** [comment]
I'll note that I'm avid to continue using flatparse, and that I'm the current maintainer for its Nix package.
